pilot/pkg/features/pilot.go

```diff
@@ -597,7 +597,7 @@ var (
 		"If false, TCP probes will not be rewritten and therefor always succeed when a sidecar is used.",
 	).Get()
 
-	EnableQUICListeners = env.RegisterBoolVar("PILOT_ENABLE_QUIC_LISTENERS", false,
```

Comment:
Aren't we supposed to change VERIFY_CERTIFICATE_AT_CLIENT default value?

Reply:
My bad, just pushed up the correction